Libraries/Components/View/View.js

```diff
@@ -14,12 +14,20 @@
 const EdgeInsetsPropType = require('EdgeInsetsPropType');
 const NativeMethodsMixin = require('NativeMethodsMixin');
 const NativeModules = require('NativeModules');
+const Platform = require('Platform');
 const React = require('React');
 const ReactNativeStyleAttributes = require('ReactNativeStyleAttributes');
 const ReactNativeViewAttributes = require('ReactNativeViewAttributes');
 const StyleSheetPropType = require('StyleSheetPropType');
 const ViewStylePropTypes = require('ViewStylePropTypes');
 
+const isTVOS = (Platform.OS === 'ios' && Platform.SystemName === 'tvOS');
```

Comment:
We're repeating this all over the place. Can you either make it a method on `Platform` or a separate function module?